src/gatsby/config.ts

```diff
@@ -1,9 +1,9 @@
 import axios from "axios";
 
 import queries from "./utils/algolia";
-import PackageRegistry from "./utils/packageRegistry";
+// import PackageRegistry from "./utils/packageRegistry";
```

Comment:
No commented-out code please. Either delete or uncomment.

Reply:
guessing this was an accident - this code cannot be removed

Reply:
whoops! this is my workaround to make gatsby build faster locally. I'm going to undo.